charts/athens-proxy/Chart.yaml

```diff
@@ -1,15 +1,17 @@
 apiVersion: v1
 name: athens-proxy
-version: 0.5.4
-appVersion: 0.11.1
+version: 0.5.5
+appVersion: v0.11.0
```

Comment:
Why the additional `v`? Is this conventional?

Reply:
because the docker tag contains the 'v'. Otherwise I would need string concatenation in the chart to add the 'v'